scripts/release_v1.js

```
@@ -0,0 +1,137 @@
+const path = require('path');
+
+const semver = require('semver');
+
+const {
+  askForConfirmation,
+  askForText,
+  checkGitStatus,
+  checkNpmAuth,
+  runExec,
+  revertCommit,
+  getDateYYYYMMDDHHMMSS,
+} = require('./release.helpers');
+
+const packageJsonPath = path.resolve(__dirname, '../next/package.json');
+const packageJson = require(packageJsonPath);
+
+async function checkGitBranchAndStatus() {
+  console.log('Checking your branch...');
+  const resultBranch = await runExec('git branch --show-current', {
+    silent: true,
+  });
+  const branchName = resultBranch.stdout.toString().trim();
+  if (branchName === 'main') {
+    console.error(`🟠 You are at "main". Are you sure you wanna release a v1 version here?`);
+    process.exit(1);
+  }
+
+  await checkGitStatus();
+}
+
+async function getNewVersion() {
+  const releaseType = process.argv[2];
+  if (!['dev', 'beta'].includes(releaseType)) {
+    console.error('🟠 Invalid release type. Use dev or beta');
+    process.exit(1);
+  }
+
+  const currentVersion = packageJson.version;
+
+  if (releaseType === 'dev') {
+    const timestamp = getDateYYYYMMDDHHMMSS();
+    console.log('Creating new dev version...');
+    return `1.0.0-dev.${timestamp}`;
+  }
+
+  // For beta releases
+  console.log('Creating new beta version...');
+  if (currentVersion.includes('-beta.')) {
+    return semver.inc(currentVersion, 'prerelease', 'beta');
+  }
+  return '1.0.0-beta.0';
```

bug: this return does not make sense. I'd throw an error instead.